src/examples/src/widgets/constrained-input/Password.tsx

```diff
@@ -0,0 +1,29 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import ConstrainedInput from '@dojo/widgets/constrained-input';
+import icache from '@dojo/framework/core/middleware/icache';
+
+const factory = create({ icache });
+
+export default factory(function Basic({ middleware: { icache } }) {
+	const value = icache.getOrSet('value', '');
+
+	return (
+		<ConstrainedInput
+			rules={{
+				length: {
+					min: 4
+				},
+				contains: {
+					atLeast: 2,
+					uppercase: 1,
+					specialCharacters: 1,
+					numbers: 1
+				}
+			}}
+			value={value}
+			onValue={(value) => icache.set('value', value)}
+			label="Enter Password"
+			type="password"
```

Comment:
How about changing this to be a `Username` example? I think having the `password` type makes it had to reason with the text you're entering in terms of providing users with an example.

Reply:
Done!